ansible/roles/bastion-ocp-version/tasks/main.yml

```
@@ -61,7 +61,7 @@
 
 - name: Extract/Download the openshift-install binary for the specific release
   shell: |
-    oc adm release extract --tools {{ ocp_release_image }}
+    oc adm release extract --tools {{ ocp_release_image }} --to {{ ocp_version_path }} --registry-config /root/.docker/config.json
```

The flag --to is not necessary as the _command's_ cwd is _ocp_version_path_
```suggestion
    oc adm release extract --tools {{ ocp_release_image }} --registry-config /root/.docker/config.json
```